### PR TITLE
feat: add userstyles to ports list

### PR DESCRIPTION
**Pull request by uncenter**

### CODE REVIEW

File: src/components/cards/Port.astro [hunk 1]
```diff
@@ -61,6 +61,7 @@ const { title, link, categories, color } = Astro.props;
 
     .port-header {
       @include utils.flex($gap: var(--space-sm));
+      flex-wrap: nowrap;
```
Comment (sgoudham) [inline src/components/cards/Port.astro:64]: Curious what the reason for this is?

**Review by sgoudham — COMMENTED** [review]
Just one comment from me!

PR looks good, can always refactor later :+1:

File: src/components/cards/Port.astro (hunk 1 above)
Comment (uncenter) [reply src/components/cards/Port.astro:64]: Prevents the port name being too long and forcing the icon down a level.

**Review by sgoudham — APPROVED** [review]

File: package.json
Comment (uncenter) [inline package.json:1]: Oop I don't know why Bun didn't resolve `simple-icons@latest` to an actual specifier but yeah we needed to update in order to include some icons.

File: src/lib/ports.ts [hunk 2]
```diff
@@ -21,20 +21,38 @@ export type Port = {
...
   upstreamed?: boolean;
 };
 
-export const yml = (await fetch("https://github.com/catppuccin/catppuccin/raw/main/resources/ports.yml")
+export type Userstyle = {
+  name: string;
+  categories: string[];
+  color: ColorName;
+  icon?: string;
+  readme: Record<string, unknown>;
```
Comment (uncenter) [inline src/lib/ports.ts:37]: This isn't the full type but it is enough for our purposes. We have the actual type definitions in each respective repository, how should we pull those in?

File: src/lib/ports.ts [hunk 3]
```diff
@@ -21,20 +21,38 @@ export type Port = {
...
+export const portsYml = (await fetch("https://github.com/catppuccin/catppuccin/raw/main/resources/ports.yml")
   .then((r) => r.text())
   .then((t) => parse(t))) as {
   ports: Record<string, Port>;
   categories: Array<Category>;
 };
 
-export const { ports, categories } = yml;
+export const userstylesYml = (await fetch(
+  "https://github.com/catppuccin/userstyles/raw/main/scripts/userstyles.yml?q=100",
```
Comment (uncenter) [inline src/lib/ports.ts:48]:
```suggestion
  "https://github.com/catppuccin/userstyles/raw/main/scripts/userstyles.yml",
```

Comment (uncenter) [reply src/lib/ports.ts:48]: Oops I forgot to remove this part, doesn't make a difference but yeah xD.

**Pull request merged**